README.rst

```diff
@@ -181,6 +181,23 @@ In the above config, if only files under `pyproject` were touched and the merge
           BUILDPIPE_STAIR_SCOPE: project
         label: 'deploy-prod pyproject :shipit:'
 
+
+Additional Features
+-------------------
+
+.. code-block:: yaml
+
+    # Only check the last commit for new changes
+    last_commit_only: true
+
+    # trigger deploy steps on master during business hours
+    deploy:
+      branch: master
+      timezone: US/Eastern
+
+The :code:`last_commit_only` flag allows you to choose between change detection in the whole history or only in the last commit that happened.
```

Comment:
Should we clarify that this only happens for the non-default branch?

Reply:
I am not sure about the differentiation between default branch and other branches anyways. Could you explain why this is there?
But yes I will update the readme

Reply:
The "default" branch is the branch to be merged. I think you only want this behavior for the "PR branch".

Reply:
Yeah I think you're right. I added some clarification to the readme now